command/hcl2_upgrade.go

```diff
@@ -437,7 +439,22 @@ func transposeTemplatingCalls(s []byte) []byte {
 		Parse(string(s))
 
 	if err != nil {
-		return fallbackReturn(err, s)
+		if strings.Contains(err.Error(), "unexpected \"\\\\\" in operand") {
```

Comment:
Nice! I like that this is only done when the parsing fails! 👏🏼

Reply:
+1 !